android/src/reactNativeVersionPatch/nativeHierarchyManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -263,7 +285,13 @@ public synchronized void manageChildren(
     }
 
     // we don't want layout animations in native-stack since it is currently buggy there
+    AnimationsManager animationsManager = ((ReaLayoutAnimator) mReaLayoutAnimator).getAnimationsManager();
     if (viewGroupManager.getName().equals("RNSScreenStack")) {
+      if (tagsToDelete == null) {
+        animationsManager.doSnapshotForTopScreenViews(viewGroup);
```

Comment:
I think want to do "snapshot *of* sth" and not "snapshot *for* sth" here? no?

Reply:
Resolved by: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/09400a87bd6e0878d84cdf6859298427905d6940